README.md

```diff
@@ -19,7 +19,7 @@ And then:
 
 ### Add new post
 
-`hugo new post/packit-100.md`
+`hugo new posts/packit-100.md`
```

Comment:
We are releasing 1.0?

Reply:
Well played, Jirko!

Reply:
No idea what you two are talking about, it's just example, but I can change it to `packit-xyz.md` if it lets you sleep.